### PR TITLE
Adding controller support and speeding build

### PR DESCRIPTION
**Pull request by matheuscmelo**
This PR adds Controller support and speeds build by removing unnecessary dependency installing

### CODE REVIEW

File: test_env/controller/controller.cfg [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+[monasca]
+monasca_endpoint = https://my.monasca.api.endpoint
+username = asperathos-monasca
+password = asperathos-monasca
+project_name = asperathos
+auth_url = https://my.monasca.authentication.url
+api_version = 2_0
+
+[k8s_replicas]
+k8s_manifest = /.kube/config
```
Comment (rafaelvfalc) [inline test_env/controller/controller.cfg]: This variable now is optional to the use of the 'k8s_replicas' plugin. There is any way to comment this information in this file to make easier to the user to be aware of this info?

File: test_env/controller/controller.cfg [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+[external_api]
+actuator_metric = cpu
+k8s_manifest = /.kube/config
```
Comment (rafaelvfalc) [inline test_env/controller/controller.cfg]: Same as above

**Review by rafaelvfalc — CHANGES_REQUESTED** [review]
This is a really good contribution, thank you a lot. I have only one comment that it is a (small) block for me. The `continuous-integration/jenkins/pr-head` has not been successful. I think that this integration test does not exists anymore. But, maybe if you 'hard push' the branch again, this unsuccessful test case will no longer appear in the PR.

**Comment by matheuscmelo** [comment]
@rafaelvfalc Thanks for the feedback, all changes addressed. Hope it's all ok now.

File: test_env/controller/controller.cfg [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+[monasca]
+monasca_endpoint = https://my.monasca.api.endpoint
+username = asperathos-monasca
+password = asperathos-monasca
+project_name = asperathos
+auth_url = https://my.monasca.authentication.url
+api_version = 2_0
+
+[k8s_replicas]
+
```
Comment (rafaelvfalc) [inline test_env/controller/controller.cfg]: Unused empty line.

**Review by rafaelvfalc — APPROVED** [review]
Just left a small comment about a unused empty line. However, this is not a block for me. Thank you for the implementation!

**Comment by rafaelvfalc** [comment]
The `continuous-integration/jenkins/pr-merge` test case has been unsuccessful. This is normal?

**Comment by matheuscmelo** [comment]
@rafaelvfalc I was configuring something in Jenkins and it made it fail. Must pass now.

**Comment by matheuscmelo** [comment]
As seen here:
http://150.165.15.68:50000/blue/organizations/jenkins/asperathos-integration-tests/detail/PR-8/7/pipeline
The build took only 4m to finish (versus 8m before).

**Comment by rafaelvfalc** [comment]
Awesome!

**Review by javanlacerda — APPROVED** [review]
Thanks! LGTM!

**Pull request merged**